src/components/Help/TooltipAndRTDLink/style.css

```
@@ -0,0 +1,3 @@
+.rtd-link {
+  color: white !important;
+}
```

```suggestion
.rtd-link,  .rtd-link:active, .rtd-link:focus, .rtd-link:hover {
  color: white;
}
```

Try to avoid using `!important`, unless absolutely necessary. Also, the link needs to be white when active, hovered, and focused.